bulk-user-setup/README.md

```diff
@@ -0,0 +1,43 @@
+The script requires following bash utilities. Please install them depending on your OS.
+
+1. jq - [Json Processor](https://stedolan.github.io/jq)
+
+2. [csvkit](https://formulae.brew.sh/formula/csvkit) - collection of CSV tools 
+
+----
+
+The script will prompt for the following information:
+
+* path to the CSV input file
+* ccd _idam-admin_ username
+* ccd _idam-admin_ password
+* idam oauth2 secret for _ccd-bulk-user-register_ client
+* environment
+
+----
+
+The CSV input file must contain the following elements, including a header row.
+
+| Header    | Description                                 |
+|-----------|---------------------------------------------|
+| email     | Email address of user                       |
+| firstName | First name of user                          | 
+| lastName  | Last name of user                           |
+| roles     | A pipe delimited list of roles for the user |
+
+> Note: The field headings are case sensitive but the order of the columns is not important.  Any additional columns will be ignored by the process.
+
+
+**Local docker testing setup**
+
+1. Create a new client using [idam admin web interface](http://localhost:8082)
+
+    * Client-id / label to  =>  "ccd-bulk-user-register"
```

Comment:
ccd-bulk-user-regist**rar**

Reply:
aha.. can we leave with that 😄 ? changing that requires two tickets (sidam, rdo)

Reply:
yep, sure. it was a minor detail already.